contents/blog/new-pricing.md

```diff
@@ -0,0 +1,144 @@
+---
+date: 2024-06-24
+title: 'PostHog is now up to 80% cheaper'
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog_is_the_cheapest_e77c4ea4a5.jpg
+author:
+  - james-hawkins
+tags:
+  - PostHog news
+---
+import {ProductScreenshot} from 'components/ProductScreenshot'
+
+export const Pricing = "https://res.cloudinary.com/dmukukwp6/image/upload/posthog_pricing_ba910c9535.jpg"
+export const PricingDark = "https://res.cloudinary.com/dmukukwp6/image/upload/posthog_pricing_dark_6b11387d6c.jpg"
+export const PricingComparison = "https://res.cloudinary.com/dmukukwp6/image/upload/posthog_pricing_comparison_f0c9e11f40.jpg"
+export const PricingComparisonDark = "https://res.cloudinary.com/dmukukwp6/image/upload/posthog_pricing_comparison_dark_ecf7d726c0.jpg"
+
+## TL;DR
```

Comment:
This section doesn't mention the downsides, which is a pretty important caveat.

Reply:
We have the draw the line somewhere. This section is already longer than I'd like. It's explained in the article clearly and I think this is enough.